### ISSUE TITLE
Discretize package will not load for SimPEG

### ISSUE THREAD

**Issue by jstults-phd** [simpeg/discretize#195]
Hello everyone, I am having a similar issue to that of user @orerocks from back in May 2019.  I am using Windows 10 machine with a conda environment running python 3.6 which I built specifically for running SimPEG.  I installed SimPEG from the conda-forge but immediately ran into issues when I tried to run the example jupyter notebooks.  If I run the code I get the warning:

`AttributeError: type object 'discretize.utils.interputils_cython.array' has no attribute '__reduce_cython__'`

If I recompile discretize using  `pip install --upgrade --force-reinstall discretize` I still run into the same issue.  Not sure where to go from here.  Using numpy version 1.16.0 for reference if that helps.

**Comment by thast** [comment]
try with `conda` instead of `pip`: `conda install -c conda-forge discretize`

**Comment by thast** [comment]
Also, if you want to use `pip`, you need the C++ compilers. From the documentation http://discretize.simpeg.xyz/en/master/content/installing.html:
![image](https://user-images.githubusercontent.com/13682747/74569014-3e44af00-4f2e-11ea-8910-f65eeb2facb5.png)

**Comment by jstults-phd** [comment]
I tried this in my original environment to no avail.  Deleted the environment, reinstalled the environment with conda 3.7 instead of conda 3.6 and still got the same error for this package.  
Discretize was installed was automatically installed when I installed simpeg using:
`conda install SimPEG --channel conda-forge` 
reinstalling discretize with conda after this simply produce the following output:
`# All requested packages already installed.`
Tried installing a fewversions of numpy (1.18.1, 1.16.1, 1.16.0) as suggested in other threads below but none seem to work.  Any advice appreciated.  What version of python do others work with SimPEG in?
https://github.com/numpy/numpy/issues/12785
https://github.com/scikit-learn-contrib/hdbscan/issues/272

**Comment by jstults-phd** [comment]
I figured out the issue.  When I tried running the .py files from the command line with my SimPEG environment activated the code ran just fine.  The issue only occurred when I was trying to run the code from a jupyter notebook.  It turned out my environment was not configured to run in jupyter notebook.  The issue is addressed in the stackoverflow thread below.  Thanks for your time.
https://stackoverflow.com/questions/37085665/in-which-conda-environment-is-jupyter-executing
